gfx.cpp

```diff
@@ -6812,6 +6812,7 @@ class GfxInternal
                             }
                         }
                     }
+                    transitionResource(sbt_buffer, D3D12_RESOURCE_STATE_NON_PIXEL_SHADER_RESOURCE);
                     command_list_->CopyBufferRegion(sbt_buffer.resource_, dst_offset, upload_buffer.resource_,
                         src_offset, upload_buffer_offset - src_offset);
```

Comment:
Doesn't look like transitioning to `D3D12_RESOURCE_STATE_COPY_DEST` is required here (or at least the debug layer doesn't complain about it).